### PR TITLE
[11.0] [MIG] maintenance_equipment_scrap

### PR DESCRIPTION
**Pull request by dalonsod**
cc @astirpe

### CODE REVIEW

File: maintenance_equipment_scrap/wizard/scrap_equipment.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                </sheet>
+                <footer>
+                    <button name="do_scrap" string="Scrap" type="object" class="oe_highlight" />
+                    <button special="cancel" string="Cancel" class="btn-default"/>
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="wizard_perform_equipment_scrap_action">
```
Comment (tbaden) [inline maintenance_equipment_scrap/wizard/scrap_equipment.xml:24]: optional change:
id before model

File: maintenance_equipment_scrap/views/maintenance_equipment_category.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="maintenance_equipment_category_view_form" model="ir.ui.view">
```
Comment (tbaden) [inline maintenance_equipment_scrap/views/maintenance_equipment_category.xml]: optional change:
use original id (hr_equipment_category_view_form)

File: maintenance_equipment_scrap/views/maintenance_equipment.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="maintenance_equipment_view_form" model="ir.ui.view">
```
Comment (tbaden) [inline maintenance_equipment_scrap/views/maintenance_equipment.xml]: optional change:
use original id (hr_equipment_view_form)

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM
If you like some optional changes

File: maintenance_equipment_scrap/__init__.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
```
Comment (astirpe) [inline maintenance_equipment_scrap/__init__.py]: Please fix the travis warning:

```
maintenance_equipment_scrap/__init__.py:1: [C8202(unnecessary-utf8-coding-comment), ] UTF-8 coding is not necessary
```

**Review by astirpe — CHANGES_REQUESTED** [review]
Tested in Runbot: works fine!

File: maintenance_equipment_scrap/__init__.py (hunk 4 above)
Comment (dalonsod) [reply maintenance_equipment_scrap/__init__.py]: Done!

File: maintenance_equipment_scrap/views/maintenance_equipment.xml (hunk 3 above)
Comment (dalonsod) [reply maintenance_equipment_scrap/views/maintenance_equipment.xml]: OK

File: maintenance_equipment_scrap/views/maintenance_equipment_category.xml (hunk 2 above)
Comment (dalonsod) [reply maintenance_equipment_scrap/views/maintenance_equipment_category.xml]: OK

File: maintenance_equipment_scrap/wizard/scrap_equipment.xml (hunk 1 above)
Comment (dalonsod) [reply maintenance_equipment_scrap/wizard/scrap_equipment.xml:24]: This suggestion actually doesn't change anything but code style, so I preferred preserve the original code in the file history

**Review by astirpe — APPROVED** [review]

**Pull request closed without merging**